src/Maybe.js

```diff
@@ -11,9 +11,15 @@ function _Just(x) {
 }
 util.extend(_Just, Maybe);
 
+_Just.prototype.isJust = R.T;
+_Just.prototype.isNothing = R.F;
```

Comment:
I think I'd prefer them both to have static properties on their prototypes rather than functions too (apologies for my earlier suggestion in #86 for using `R.T`/`R.F`). We still have the functions available on `Maybe`/`Either` for people to use where a function is more appropriate.